src/index.html

```diff
@@ -22,6 +22,327 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo-tag"
+        href="#"
+      >
+        <img
+          class="header__logo-image"
+          src="./images/logo.png"
+          alt="MOYO_LOGO"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__list-link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card">
+      <div
+        class="card__container"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Imac A1419"
+          class="card__container-image"
+        />
+        <h2 class="card__container-title">
```

Comment:
Camille, there isn't nothing wrong with this name, ok? But it could be simplified a little.
What you are saying is: "Container Title" is an _Element_ of the _Block_ "Card". And this is ok.
But I believe you mistakenly thought that you needed to add "__container" to every element's name, what isn't the case.
The name "card__title" would work just as fine, and this is valid for all following elements ;D